docs/admin-guide/provisioning/vsphere.md

```diff
@@ -149,33 +214,41 @@ workers:
     count: 1
     cpu: 2
     memory: 4
-    disk: 10
-    template: k8s-1.16.4
+    disk: 20
+    template: k8s-v1.18.6-template
 ```
 
-## 5. Provision the cluster
+## 7. Provision the cluster
 
 Provision the cluster with:
 
 ```bash
 karina provision vsphere-cluster -c cluster.yaml
 ```
 
-## 6. Deploy a CNI
```

Comment:
Maybe merge these steps into a deploy all, with a note for how they can be run individually